tests/keras_tests/feature_networks_tests/feature_networks/weights_mixed_precision_tests.py

```diff
@@ -221,6 +223,54 @@ def compare(self, quantized_model, float_model, input_x=None, quantization_info=
             "final weights and activation memory sum should be equal to total memory.")
 
 
+class MixedPercisionSearchKPI4BitsAvgTestCombinedNMS(MixedPercisionBaseTest):
+    def __init__(self, unit_test, layer_type):
+        assert layer_type in ['combined_nms', 'SSDPostProcess']
+        self.layer_type = layer_type
+        super().__init__(unit_test)
+
+    def get_kpi(self):
+        # kpi is for 4 bits on average
+        return KPI(17920 * 4 / 8)
+
+    def create_networks(self):
+        inputs = layers.Input(shape=self.get_input_shapes()[0][1:])
+        x = layers.Conv2D(32, 4)(inputs)
+        x = layers.BatchNormalization()(x)
+        x = layers.Conv2D(32, 4)(x)
+        x = layers.ReLU()(x)
+        if self.layer_type == 'combined_nms':
```

Comment:
maybe create two tests one for each of the options? (more "unit-testy")

Reply:
separated the tests